qemu/tests/cfg/blockdev_mirror_qemuio_ready_job.cfg

```diff
@@ -0,0 +1,27 @@
+- blockdev_mirror_qemuio_ready_job:
+    only Linux
+    only filesystem iscsi_direct ceph nbd gluster_direct
```

Comment:
remove 'iscsi_direct ceph nbd gluster_direct' if we don't want to cover these storage backends.

Reply:
Have updated, thanks @zhencliu